utilities/application.sh

```diff
@@ -96,6 +96,7 @@ run_command_silent_unless_verbose() {
     shift
 
     if argsparse_is_option_set "verbose"; then
+        echo "${command}" "${@}"
```

Comment:
Leftover?

Reply:
Its OK. A general comment though is to please split changes into more commits. You always just dump a handful of random changes in at once so its impossible to know what your intentions are.